pkg/proto/datadog/remoteconfig/remoteconfig.proto

```diff
@@ -86,6 +86,7 @@ message ClientTracer {
   string language = 2;
   string tracer_version = 3;
   string service = 4;
+  repeated string extra_services = 8;
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 Nitpick:** ‏`additional_services` ?

Reply:
no.